test/IFrameNavigatorTests.ts

```diff
@@ -657,6 +657,103 @@ describe("IFrameNavigator", () => {
       });
     });
 
+    it("should show the default SVG logo when custom LibraryIcon is an empty string", async () => {
```

Comment:
So based on these tests, it still seems to work really strangely to me.  1) Why do we have to pass in an empty string for libraryIcon?  If it's optional, we should be able to pass in 
``` upLink: {
          url: new URL("http://up.com"),
          label: "Up Text",
          ariaLabel: "Up Aria Text",
        }
```

and just have it handle it.

Reply:
D'oh.  This was what you had before.  OK sorry, I think I talked myself in a circle.